.github/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+bancho.py is in a volatile state, therefore we only provide support for the latest version. It is the responsibility of the user to remain up-to-date.
+
+## Reporting a Vulnerability
+
+If you notice any vulnerabilities, pleae contact us urgently on
+[Discord](https://discord.gg/ShEQgUx) or via security advisories in the repository.
```

Comment:
would prefer email directly for reporting security issues - cmyuiosu@gmail.com

Reply:
sure